var/spack/repos/builtin/packages/vampirtrace/package.py

```diff
@@ -0,0 +1,63 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Vampirtrace(AutotoolsPackage):
+    """VampirTrace is an open source library that allows detailed logging of
+    program execution for parallel applications using message passing (MPI)
+    and threads (OpenMP, Pthreads)."""
+
+    homepage = "https://tu-dresden.de/zih/forschung/projekte/vampirtrace"
+    url      = "http://wwwpub.zih.tu-dresden.de/~mlieber/dcount/dcount.php?package=vampirtrace&get=VampirTrace-5.14.4.tar.gz"
+
+    version('5.14.4', '1c92b23169df9bcc860e5fc737dbc9c9')
+
+    variant('mpi', default=True, description='Enable MPI support')
+
+    depends_on('mpi', when='+mpi')
+    depends_on('otf')
+    depends_on('papi')
+    depends_on('zlib')
+
+    def configure_args(self):
+        spec = self.spec
+        compiler = self.compiler
+
+        args = [
+            '--with-extern-otf',
+            '--with-extern-otf-dir={0}'.format(spec['otf'].prefix),
+            '--with-papi-dir={0}'.format(spec['papi'].prefix),
+            '--with-zlib-dir={0}'.format(spec['zlib'].prefix)
+        ]
+
+        env['CC'] = compiler.cc
+        env['CXX'] = compiler.cxx
+        env['FC'] = compiler.fc
```

Comment:
In that case you would want to filter the compiler wrappers after installation. See any of the MPI packages for how to do this. We still need the build to be done with the compiler wrappers though.